src/metabase/sync/analyze/fingerprint.clj

```diff
@@ -212,18 +212,14 @@
     log-progress-fn :- LogProgressFn]
    (fingerprint-fields-for-db!* database tables log-progress-fn (constantly true)))
 
-  ;; TODO: Maybe the driver should have a function to tell you if it supports fingerprinting?
   ([database        :- i/DatabaseInstance
     tables          :- [:maybe [:sequential i/TableInstance]]
     log-progress-fn :- LogProgressFn
     continue?       :- [:=> [:cat ::FingerprintStats] :any]]
    (qp.store/with-metadata-provider (u/the-id database)
      (reduce (fn [acc table]
                (log-progress-fn (if *refingerprint?* "refingerprint-fields" "fingerprint-fields") table)
-               (let [results (if (= :googleanalytics (:engine database))
-                               (empty-stats-map 0)
-                               (fingerprint-fields! table))
-                     new-acc (merge-with + acc results)]
+               (let [new-acc (merge-with + acc (fingerprint-fields! table))]
                  (if (continue? new-acc)
```

Comment:
Out of scope nitpick: I dislike how we mix mapping, summing, and prematurely terminating all within the reducing function. I think we could clean this up with`transduce`?